cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml

```diff
@@ -48,6 +48,12 @@
   <logger name="org.apache.twill" level="INFO"/>
   <logger name="org.apache.twill.internal.kafka.client.SimpleKafkaConsumer" level="WARN"/>
   <logger name="co.cask.cdap" level="INFO"/>
+  <!--
+    The netty http library logs an ERROR always if it is raised
+    not from user handler, which usually is due to network error.
+    Disabling the log to avoid flooding.
```

Comment:
the comment is not clear. Then why would we not suppress a netty class logger?

Reply:
I don't get this. The netty http class logger is being suppressed by setting level to OFF.

Reply:
nvm